backport/createStatusComment.ts

```diff
@@ -23,14 +24,20 @@ export const getCommentFromResponse = (pullNumber: number, backportResponse: Bac
 
   const helpParts = [];
 
-  if (hasAnySuccessful) {
-    helpParts.push('Successful backport PRs will be merged automatically after passing CI.');
+  if (hasAllSuccessful) {
+    helpParts.push('The backport PRs will be merged automatically after passing CI.');     
+  } else if (hasAnySuccessful) {
+    if (backportResponse.results.length === 1) {
+      helpParts.push('This backport PR will be merged automatically after passing CI.'); 
+    } else {
+      helpParts.push('Successful backport PRs will be merged automatically after passing CI.');
+    }
```

Comment:
This is just nice-to-haves. I don't know if it's too much having this many conditions for such a simple message. I just got a little confused when I saw this

![image](https://user-images.githubusercontent.com/209966/110705939-6406bb00-81f7-11eb-8735-b21dbdf1fd2c.png)

In that case it seemed simpler to say "This backport will be merged automatically"